docs/user-guide/local.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: main
+title: Local Build
+category: User Guide
+menu: menu
+toc:
+- title: What is sd-local ?
+  url: "#what-is-sd-local-?"
+- title: Installing sd-local
+  url: "#installing-sd-local"
+- title: Simple usage
+  url: "#simple-usage"
+- title: config command
+  url: "#config-command"
+- title: build command
+  url: "#build-command"
+---
+
+# What is sd-local ?
+sd-local enable you to build as Screwdriver does without uploading source code to SCM.
+You can use features including Command, Template, Metadata and so on as same as Screwdriver.
+
+
+# Installing sd-local
+## Requirements
+- [docker](https://www.docker.com/)
+  - Use Docker container as build environment.
+- [git](https://git-scm.com/)
+  - Required when using [--src-url] option.
+
+
+## How to Install
+
+
+You can install sd-local by the following command.
+```bash
+$ go get github.com/screwdriver-cd/sd-local
+```
+
+
+Note:
+go command is required to install sd-local.
+https://golang.org/
```

Comment:
Can this section be combined with other Requirements?

Reply:
I fixed.